Lesson16/fercan/script.js

```diff
@@ -0,0 +1,100 @@
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
```

Comment:
In this exercise, you're asked to write a function, so you need to start with a function definition, e.g.:
```

function compareNumbers(number1, number2) {
// if statement should be here inside the function
}
```

In our case, 'asks the user to enter two numbers' means that the function should take 2 parameters (number1, number2) as added in the example above. 

After function definition is created, it's time to call the function with actual numbers:
```
compareNumbers(2, 10);
compareNumbers(-20, 0);
compareNumbers(23, 3);

```